packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -34,7 +33,9 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match
         <Loader />
       ) : (
         <>
-          <div className={styles.innerChartContainer}>
+          <div
+            className={styles.innerChartContainer}
+            style={{ paddingRight: windowSize.size === 'normal' && openWindows > 0 && '0' }}>
```

Comment:
Right now `paddingRight` is `false` or `0`. I would be more happy if both are legit values (e.g., `auto` and `0`).